.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 .settings
 *.log
 dist/
+.idea
```

Comment:
Please remove this. This should be handled by developer's global gitignore.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore